app/view-plants/page.tsx

```diff
@@ -179,16 +180,35 @@ export default function Page() {
           </TopRowContainer>
           {viewingOption === 'myPlants' && (
             <div>
+              <div className="plantSelection">
```

Comment:
is `plantSelection` this a valid style? This should be a styled component instead 

Also, to reduce code redundancy, the entire PlantSelection should be moved outside of each viewing option section.

also, we just decided that the header should not appear if the user isn't logged in/not onboarded, so separating it out will also help with handling that case!